### ISSUE TITLE
Caja-Dropbox: use_enable() calls are not allowed in global scope

### ISSUE THREAD

**Issue by NHOrus** [oz123/mate-de-gentoo#24]
Attempts to install it fail on dependency check, with following error:

```
ERROR: mate-extra/caja-dropbox-1.14.0::mate-de-gentoo failed (depend phase):
 *   use_enable() calls are not allowed in global scope
 * 
 * Call stack:
 *                    ebuild.sh, line 606:  Called source '/var/lib/layman/mate-de-gentoo/mate-extra/caja-dropbox/caja-dropbox-1.14.0.ebuild'
 *   caja-dropbox-1.14.0.ebuild, line  42:  Called use_enable 'debug'
 *                    ebuild.sh, line  78:  Called die
 * The specific snippet of code:
 *          eval "${x}() { die \"\${FUNCNAME}() calls are not allowed in global scope\"; }"
 * 
 * If you need support, post the output of `emerge --info '=mate-extra/caja-dropbox-1.14.0::mate-de-gentoo'`,
 * the complete build log and the output of `emerge -pqv '=mate-extra/caja-dropbox-1.14.0::mate-de-gentoo'`.
 * Working directory: '/usr/lib64/python2.7/site-packages'
 * S: '/var/tmp/portage/mate-extra/caja-dropbox-1.14.0/work/caja-dropbox-1.14.0'
```

Same with caja-dropbox-1.13.0

**Comment by oz123** [comment]
Can you specify which version of portage and layman are you using?

I am using this repository directly with portage, I can't reproduce your error.

**Comment by NHOrus** [comment]
sys-apps/portage-2.3.0::gentoo
app-portage/layman-2.4.1-r1

I am on ~amd64

**Comment by oz123** [comment]
OK, this ebuild is definitely wrong if you use portage 2.3 or later. I will try and fix this asap.

**Comment by oz123** [comment]
This is now fixed in caja-dropbox-1.14.0-r1

**Comment by NHOrus** [comment]
This is now came back in caja-dropbox-1.12.0::mate-de-gentoo
